packages/gatsby-theme-carbon/gatsby-config.js

```diff
@@ -90,6 +91,9 @@ module.exports = (themeOptions) => {
             ...defaultRemarkPlugins,
             ...gatsbyRemarkPlugins,
           ],
+          remarkPlugins: [
+            ...remarkPlugins,
+          ],
```

Comment:
I think we can just pass this through for now since we're not adding any defaults of our own.
```suggestion
          remarkPlugins,
```

Reply:
Done!